modelling/src/test/java/org/axonframework/modelling/saga/AnnotatedSagaTest.java

```diff
@@ -57,7 +57,6 @@ void setUp() {
         );
     }
 
-    @Test
```

Comment:
Why did we drop this annotation here? I would assume this is still a valid test to run.

Reply:
I have no clue why this was part of the pr, I reverted the changes made to this class.